lib/acts_as_list/active_record/acts/list.rb

```diff
@@ -262,11 +262,12 @@ def higher_item
 
         # Return the next n higher items in the list
         # selects all higher items by default
-        def higher_items(limit=nil)
+        def higher_items(limit = nil)
```

Comment:
I believe this is causing your test failures. I'm not sure spaces are ok between the argument and the default value. So do `limit=nil`.